src/client/components/utils/featureToggles.ts

```diff
@@ -0,0 +1,7 @@
+import { Market, useMarket } from './CurrentLocale'
+
+export const useCreditCheckInfo = () => {
```

Comment:
This is really not important but maybe this doesn't need to be a hook but it could take the market as a parameter instead and just be a regular ol' function? Then it could be named something like `hasCreditCheckInfo`.

Reply:
I considered that as well, even have it as a plain object as 
```
export const hasCreditCheckInfo = {
   [Market.No]: true
}
```
but that forces the `useMarket`-usage onto the parent component,  imo it's better to have it like this, to be used without dependencies 🤷‍♂️

Reply:
Sure, I don't feel super strongly about this 😊  (the `useMarket` function is on its way out though, but changing this function to the new ways will be easy so that really doesn't matter). 

But one other thing is that I think this file should probably be in the `/src/client/utils` folder instead of under `/components` 🤔